req.go

```diff
@@ -66,3 +68,10 @@ func DomainName(x string) func(*Req) {
 		req.DomainName = x
 	}
 }
+
+// Set request ID
+func setRequestID(x string) func(*Req) {
```

Comment:
To be in line with the other modifier functions should we name it just `RequestID()`?